pkg/apis/aerogear/v1alpha1/unifiedpushserver_types.go

```diff
@@ -21,6 +21,9 @@ type UnifiedPushServerStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	PostgresqlReady  bool `json:"postgresqlReady"`
+	UnifiedpushReady bool `json:"unifiedpushReady"`
```

Comment:
If these are actual status fields that we want, how are the names/structure?

Possible alternative:

```
ready:
  postgresql: true
  unifiedpush: true
```

Reply:
normally speaking, this is up to the maintainer to decide what data should be put in the `status` field. But I think there should be at least a single field to indicate if the service/application is ready or not. So I would prefer:

```
status: prepare|provisioning|running|error
```
so that if I need to check if the service is ready to use, I can just check this field. With the current implementation, I have to check both fields but that is very specific to the application itself and I don't really care which component is ready or not.

Reply:
Cool, thanks! That's what I was mostly unclear about! :)